### PR TITLE
MEN-4180: improve onboarding tooltips visibility

### PR DESCRIPTION
**Pull request by tranchitella**

### CODE REVIEW

**Review by mzedel — CHANGES_REQUESTED** [review]
👍 nicely done, the fallback approach is just what I had in mind! if I remember correctly there were a few more steps that served as an anchor though (or we will have to reopen the issue very soon)...

File: src/js/actions/onboardingActions.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ export const getOnboardingState = () => (dispatch, getState) => {
       acceptedDevices.length && store.devices.byId[acceptedDevices[0]].hasOwnProperty('attributes')
         ? store.devices.byId[acceptedDevices[0]].attributes.device_type
         : null;
-    const progress = onboardingState.progress || determineProgress(acceptedDevices, pendingDevices, releases, pastDeployments);
+    const progress = applyOnboardingFallbacks(onboardingState.progress) || determineProgress(acceptedDevices, pendingDevices, releases, pastDeployments);
```
[the remark this answers is not in the document]
Comment (tranchitella) [reply src/js/actions/onboardingActions.js:34]: Ack

**Comment by tranchitella** [comment]
@mzedel ready for review; I've incorporated your changes to the fallback, but also added three commits (you can review them, specifically) to streamline the onboarding tips. It works well here, now. No more spurious welcomeTip at application startup, which was annoying me so much.

**Review by mzedel — APPROVED** [review]
Thank you very much for the fixes! 🎉 especially for the welcometip!

**Comment by mzedel** [comment]
merging to get this straight to staging...

**Pull request merged**